src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -129,11 +130,21 @@ class LedgerMasterImp
         , mValidLedgerClose (0)
         , mValidLedgerSeq (0)
         , mBuildingLedgerSeq (0)
-        , standalone_ (standalone)
-        , fetch_depth_ (getApp().getSHAMapStore().clampFetchDepth (fetch_depth))
-        , ledger_history_ (ledger_history)
-        , ledger_fetch_size_ (ledger_fetch_size)
-    {
+        , standalone_ (config.RUN_STANDALONE)
+        , fetch_depth_ (getApp ().getSHAMapStore ().clampFetchDepth (config.FETCH_DEPTH))
+        , ledger_history_ (config.LEDGER_HISTORY)
```

Comment:
Yes, intentional. Instead of passing the params separately into the LedgerMasterImp ctor, we now pass a Config reference.